beacon-chain/core/blocks/spectest/block_header_mainnet_test.go

```diff
@@ -5,5 +5,6 @@ import (
 )
 
 func TestBlockHeaderMainnet(t *testing.T) {
+	t.Skip()
```

Comment:
Why? Add a reason like `t.Skip("Disabled until X is implemented")`

Reply:
Perfect. Thanks for the suggestion